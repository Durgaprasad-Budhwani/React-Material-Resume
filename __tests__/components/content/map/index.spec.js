/**
 * Created by durgaprasad on 10/16/17.
 */
/* global expect,it */
import React from 'react';
import MapContainer from '../../../../src/components/content/map';
import renderer from 'react-test-renderer';

it('renders map', () => {
  const tree = renderer.create(
    <MapContainer />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});