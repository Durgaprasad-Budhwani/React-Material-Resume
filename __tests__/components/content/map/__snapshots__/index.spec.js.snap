// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders map 1`] = `
<div
  className="MuiGrid-typeContainer-4 MuiGrid-spacing-xs-16-25 MuiGrid-direction-xs-column-6 MapContainer-container-1"
>
  <div
    className="MuiGrid-typeItem-5 MapContainer-contactContainer-2"
  >
    <div
      className="MuiGrid-typeContainer-4 MuiGrid-typeItem-5 MuiGrid-spacing-xs-24-26 MuiGrid-direction-xs-column-6"
    >
      <div
        align="center"
        className="MuiGrid-typeContainer-4 MuiGrid-typeItem-5 MuiGrid-spacing-xs-24-26 MuiGrid-justify-xs-center-20"
      >
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-3-31 ContactInfo-profileListItem-94"
        >
          Address:
        </div>
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-9-37 ContactInfo-profileListItemValue-95"
        >
          Pune, India
        </div>
      </div>
      <div
        align="center"
        className="MuiGrid-typeContainer-4 MuiGrid-typeItem-5 MuiGrid-spacing-xs-24-26 MuiGrid-justify-xs-center-20"
      >
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-3-31 ContactInfo-profileListItem-94"
        >
          Email:
        </div>
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-9-37 ContactInfo-profileListItemValue-95"
        >
          <a
            className="ContactInfo-profileValueAItemValue-96"
            href="mailto:durgaprasad.budhwani@gmail.com"
          >
            durgaprasad.budhwani@gmail.com
          </a>
        </div>
      </div>
      <div
        align="center"
        className="MuiGrid-typeContainer-4 MuiGrid-typeItem-5 MuiGrid-spacing-xs-24-26 MuiGrid-justify-xs-center-20"
      >
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-3-31 ContactInfo-profileListItem-94"
        >
          Phone:
        </div>
        <div
          className="MuiGrid-typeItem-5 MuiGrid-grid-xs-9-37 ContactInfo-profileListItemValue-95"
        >
          <a
            className="ContactInfo-profileValueAItemValue-96"
            href="tel:+91-9545251359"
          >
            +91-9545251359
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-typeItem-5 MuiGrid-grid-xs-28 MapContainer-mapContainer-3"
  >
    <div />
  </div>
</div>
`;
